website/auth_pages.py

```diff
@@ -146,6 +146,15 @@ def signup(self):
         if "country" in body:
             if not body["country"] in COUNTRIES:
                 return gettext("country_invalid"), 400
+        if "heard_about" in body:
+            if isinstance(body["heard_about"], str):
+                body["heard_about"] = [body["heard_about"]]
+            if not isinstance(body["heard_about"], list):
+                return gettext("heard_about_invalid"), 400
+            for language in body["heard_about"]:
+                if language not in ["from_another_teacher", "social_media", "from_video", "from_magazine_website", \
```

Comment:
Please use a more explanatory variable name than `language`, maybe `option`?

Reply:
Completely agree! Don't know why I didn't change it before the PR. Anyways, applied changes!